files/image_config/copp/copp_cfg.j2

```diff
@@ -59,8 +59,47 @@
 		    "red_action": "drop",
 		    "trap_action": "trap",
 		    "trap_priority": "1"
-
-	    }
+	    }{%- if include_p4rt == "y" %},
+	    "trap.group.cpu.queue.1": {
```

Comment:
can we please rename this to the existing format? `cpu_queue1_group1`, `cpu_queue2_group1`. We don't need `trap.group` prefix in the names here